assets/sass/_container.scss

```diff
@@ -0,0 +1,52 @@
+// scss-lint:disable IdSelector
+#okta-sign-in {
+// scss-lint:enable IdSelector
+
+  // Html and body styles in _base.scss that were moved here
+  font-family: $fonts;
+  font-size: $font-size;
+  font-weight: $font-weight-body;
+  line-height: 1.4;
+  background-color: $secondary-bg-color;
```

Comment:
Hmm, so some of these don't make sense here (and in general, will just need to be ported to okta-core), i.e:
- Is background-color here for the container, or for the background page?
- min-height - we're setting the height below (and don't really want the widget container to be 100%)
- position: relative and overflow: auto were fixes on the container for embedded scenarios - probably don't belong here, and will definitely need to be tested in okta-core when we port these over